test/functions/priceAbsoluteChangeTest.js

```diff
@@ -0,0 +1,74 @@
+require('../helper.js')
+
+describe('priceAbsoluteChange_', () => {
+  const slug = 'santiment'
+  const from = '2019-01-01T00:00:00.000Z'
+  const to = '2019-01-03T23:59:59.000Z'
```

Comment:
If we use only `Date`(without time) what will be the time set for the `from` and `to` params? Is it always 00:00:00, or the `to` will be set to `23:59:59`? If it's not the second case - then the test does not represent a real scenario

Reply:
Good catch, fixed. 
```js
  const from = '2019-01-01T00:00:00.000Z'
  const to = '2019-01-03T00:00:00.000Z'
```

Reply:
In such case I would rather expect that the `from` and `to` are both inclusive, but in this case `from` is inclusive and `to` is exclusive

Reply:
This is the current behavior of the API. I have executed a graphQL query, copied and modified the response into the test.